packages/rpc-core/src/rpc-subscriptions/program-notifications.ts

```diff
@@ -0,0 +1,75 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+
+import {
+    AccountInfoBase,
+    AccountInfoWithBase58Bytes,
+    AccountInfoWithBase58EncodedData,
+    AccountInfoWithBase64EncodedData,
+    AccountInfoWithBase64EncodedZStdCompressedData,
+    AccountInfoWithJsonData,
+    Commitment,
+    RpcResponse,
+    U64UnsafeBeyond2Pow53Minus1,
+} from '../rpc-methods/common';
+
+type ProgramNotificationsMemcmpFilter = Readonly<{
+    offset: U64UnsafeBeyond2Pow53Minus1;
+    bytes: string;
+    encoding: 'base58' | 'base64';
+}>;
+
+type ProgramNotificationsDatasizeFilter = Readonly<{
+    dataSize: U64UnsafeBeyond2Pow53Minus1;
```

Comment:
Kinda crazy that this is actually a `u64`.

Reply:
Def overkill 😂